Description: Symbolic link in build target instead of hard link install
 For packaging it is preferable to generate a symbolic link for ldap2dnsd
 that can be copied, instead of hard linking during the install target.
 .
 Patch not yet forwarded to upstream.
--- a/Makefile
+++ b/Makefile
@@ -35,7 +35,7 @@ ldap2dns: ldap2dns.o
 	$(LD) $(LDFLAGS) $(LIBS) -o $@ $+
 
 ldap2dnsd: ldap2dns
-	ln -f ldap2dns ldap2dnsd
+	ln -sf ldap2dns ldap2dnsd
 
 ldap2dns-dbg: ldap2dns.o-dbg
 	$(LD) $(LDFLAGS) $(LIBS) -o $@ $+
@@ -51,8 +51,9 @@ install: all
 	mkdir -p $(INSTALL_PREFIX)/$(LDAPCONFDIR)/schema
 	mkdir -p $(INSTALL_PREFIX)/$(MANDIR)/man1
 	install -s -m 755 ldap2dns $(INSTALL_PREFIX)/$(PREFIXDIR)/bin/
-	ln -f $(INSTALL_PREFIX)/$(PREFIXDIR)/bin/ldap2dns \
-		$(INSTALL_PREFIX)/$(PREFIXDIR)/bin/ldap2dnsd
+# Debain Maintainer: Use symbolic link file in build target instead
+#	ln -f $(INSTALL_PREFIX)/$(PREFIXDIR)/bin/ldap2dns \
+#		$(INSTALL_PREFIX)/$(PREFIXDIR)/bin/ldap2dnsd
 	install -m 755 ldap2tinydns-conf $(INSTALL_PREFIX)/$(PREFIXDIR)/bin/
 	install -m 644 ldap2dns.schema $(INSTALL_PREFIX)/$(LDAPCONFDIR)/schema/
 	install -m 644 ldap2dns.1 $(INSTALL_PREFIX)/$(MANDIR)/man1
